# HG changeset patch
# User Peter Moore <pmoore@mozilla.com>
# Date 1548924901 -3600
#      Thu Jan 31 09:55:01 2019 +0100
# Node ID 7aabec1a6817a1256c90544bb8286eae69c21083
# Parent  80d449d8ad1dbbc91ddfbca4547d564a6d380315
Testing generic-worker  on nss-win2012r2-new worker type; try: -p win32,win64 -t none -u all

diff --git a/automation/taskcluster/graph/src/extend.js b/automation/taskcluster/graph/src/extend.js
--- a/automation/taskcluster/graph/src/extend.js
+++ b/automation/taskcluster/graph/src/extend.js
@@ -830,7 +830,7 @@ async function scheduleTestBuilds(name, 
 
 async function scheduleWindows(name, base, build_script) {
   base = merge(base, {
-    workerType: "nss-win2012r2",
+    workerType: "nss-win2012r2-new",
     env: {
       PATH: "c:\\mozilla-build\\bin;c:\\mozilla-build\\python;" +
 	    "c:\\mozilla-build\\msys\\local\\bin;c:\\mozilla-build\\7zip;" +
